packages/shared/src/utils/days.spec.ts

```diff
@@ -33,6 +33,8 @@ describe('days', () => {
       expect(binaryDaysToDateArray(DaysOfTheWeek.Sunday | DaysOfTheWeek.Monday | DaysOfTheWeek.Tuesday | DaysOfTheWeek.Wednesday
       | DaysOfTheWeek.Thursday | DaysOfTheWeek.Friday | DaysOfTheWeek.Saturday)).to.eql([1, 2, 3, 4, 5, 6, 7]);
     });
+    it('expect to return next day Sunday to Monday', () => {
+      expect(binaryDaysToDateArray(nextDay(DaysOfTheWeek.Sunday))).to.eql([2]);
```

Comment:
[2] can be replaced with [DaysOfTheWeek.Monday]

Reply:
There are couple of more untested functions. I am not sure what the goal is but wonder if you plan to write tests for them.

Reply:
This was just working with the existing unit test you'd provided as an example-- I must have kept this change when I forked to work on the IT/unit tests.

Reply:
Updated